### ISSUE TITLE
Travis Clang?

### ISSUE THREAD

**Issue by ChrisRackauckas** [JuliaDiffEq/DASSL.jl#10]
Is there a reason why you have the Travis tests using the Clang compiler instead of the normal Travis setup? You don't have any binary dependencies so it seems very unnecessary. Also, release is kept as v0.4 now, should set the environment separately. I could put this to a standard Travis script if you want, but I just want to double check that you aren't using the compiler for something special.

**Comment by pwl** [comment]
Probably no reason, I started this package a long time ago (maybe even in the times of Julia 0.2) and I think I just copied the Travis file from somewhere else or generated it automatically.  It should definitely be updated to the current standards.  I still have to register the new version of this package properly, v0.0.5 got declined in the METADATA because of inconsistencies in the REQUIRE file.  I think I should regenerate all the meta files using PkgDev.
